### ISSUE TITLE
Vesicoureteral Reflux Grade I

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6825]
**Preferred term label:**
Vesicoureteral reflux grade I

**Synonyms**
VUR grade I, VUR I

**Definition (free text, please give PubMed ID)**
Vesicoureteral reflux into the ureter only.

**Parent term (use hpo.jax.org/app)**
Low-grade vesicoureteral reflux (newly proposed term)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
